adapter/PreciceInterface.h

```diff
@@ -23,7 +23,7 @@
  *  Displacements - FSI data to be written (by the Calculix adapter)
  *  DisplacementDeltas - FSI data to be written (by the Calculix adapter)
  */
-enum CouplingDataType {TEMPERATURE, HEAT_FLUX, CONVECTION, FORCES, DISPLACEMENTS, DISPLACEMENTDELTAS};
+//enum CouplingDataType {TEMPERATURE, HEAT_FLUX, CONVECTION, FORCES, DISPLACEMENTS, DISPLACEMENTDELTAS};
```

Comment:
Your change concerning the position of the `CouplingDataType` enumeration is confirmed, but please also delete the deprecated code here and don't leave it commented out.